### PR TITLE
Improve user guide documentation

### PR DESCRIPTION
**Pull request by TheBeatles1994**

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -5,8 +5,17 @@ This is the user guide for Open-Local.
 ## Requirements
 
 - Kubernetes v1.18+
-- helm v3.0+
+- Helm v3.0+
```
Comment (peter-wangxu) [inline docs/user-guide.md:8]: also include Linux distro we tested, and critical rpms like lvm2

File: docs/user-guide.md [hunk 2]
```diff
@@ -5,8 +5,17 @@ This is the user guide for Open-Local.
...
 - [lvm2](https://en.wikipedia.org/wiki/Logical_Volume_Manager_(Linux))
+- At least one block device
+
+## Configuring
+
+You'd better provide one block device for **each node**, which will be used for creating VolumeGroup by Open-Local.
```
Comment (peter-wangxu) [inline docs/user-guide.md:14]: Avoid using second-person pronouns in written language, eg
> to create a Volume Group by Open Local automatically, edit xxxx

File: docs/user-guide.md [hunk 3]
```diff
@@ -47,10 +56,19 @@ NAME       STATE       PHASE     AGENTUPDATEAT   SCHEDULERUPDATEAT   SCHEDULERUP
 minikube   DiskReady   Running   30s             0s
 ```
 
-Modify the requested spec.resourceToBeInited.vgs of the custom resource to create a VG with block device `/dev/vdb` on related node.
+You can check if volume group is created and managed successfully by Open-Local by command as follows:
```
Comment (peter-wangxu) [inline docs/user-guide.md:59]: Ditto
> Use following command to check ...

**Review by peter-wangxu — COMMENTED** [review]
Good work, some minor comments

File: docs/user-guide.md (hunk 1 above)
Comment (TheBeatles1994) [reply docs/user-guide.md:9]: done

File: docs/user-guide.md (hunk 2 above)
Comment (TheBeatles1994) [reply docs/user-guide.md]: done

File: docs/user-guide.md (hunk 3 above)
Comment (TheBeatles1994) [reply docs/user-guide.md]: done

**Review by peter-wangxu — APPROVED** [review]
LGTM

**Pull request merged**